frontend/src/components/tree/ImageSlider.tsx

```diff
@@ -0,0 +1,109 @@
+import { Splide, SplideSlide } from "@splidejs/react-splide";
+import "@splidejs/react-splide/css";
+import { ArrowRight, ArrowLeft } from 'lucide-react';
+import React, { useRef, useState, useEffect } from 'react';
+import SplideInstance from '@splidejs/splide';
+
+interface ImageSliderProps {
+  images: string[];
+}
+
+const ImageSlider: React.FC<ImageSliderProps> = ({ images }) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [visibleImages, setVisibleImages] = useState(4);
+
+  const options = {
+    type: 'slide',
+    perPage: visibleImages,
+    perMove: 1,
+    gap: '1rem',
+    pagination: false,
+    arrows: false,
+  };
+
+  const splideRef = useRef<SplideInstance | null>(null);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth >= 1280) {
+        setVisibleImages(4);
+      } else if (window.innerWidth >= 1024) {
+        setVisibleImages(3);
+      } else if (window.innerWidth >= 768) {
+        setVisibleImages(2);
+      } else {
+        setVisibleImages(1);
+      }
+    };
+
+    handleResize();
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
+  const handleMoved = (splide: SplideInstance) => {
+    setCurrentIndex(splide.index);
+  };
+
+  return (
+    <div className="relative w-full">
+      <h2 className="font-bold text-xl mb-4">Hinterlegte Bilder:</h2>
+
+      <Splide
+        options={options}
+        aria-label="Image Slider"
+        ref={splideRef as React.RefObject<SplideInstance>}
+        onMoved={handleMoved}
+        className="w-full"
+      >
+        {images.map((image, index) => (
+          <SplideSlide key={index}>
+              <img
+                src={image}
+                alt={`Slide ${index + 1}`}
+                className="w-full aspect-square object-cover rounded-lg shadow-lg"
+              />
+          </SplideSlide>
+        ))}
+      </Splide>
+
+      {images.length > 1 && (

```

Comment:
Please check the [docs](https://splidejs.com/integration/react-splide/) of the slider. 

The navigation elements should also be used from the slider package itself.

Reply:
The navigation elements of splide look different than the one in Figma. Thats why I wrote that part. When I use the Splide arrows it looks like this:
![Unbenannt](https://github.com/user-attachments/assets/ebfef6f9-1bf0-4348-8d26-fda4301e64ee)

Is this okay?